src/Core/Content/Seo/SeoUrlTwigFactory.php

```diff
@@ -0,0 +1,36 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Content\Seo;
+
+use Cocur\Slugify\Bridge\Twig\SlugifyExtension;
+use Cocur\Slugify\SlugifyInterface;
+use Twig\Environment;
+use Twig\Extension\EscaperExtension;
+use Twig\Loader\ArrayLoader;
+
+class SeoUrlTwigFactory
+{
+    public static function createTwigEnvironment(SlugifyInterface $slugify, iterable $twigExtensions = []): Environment
```

Comment:
Why needs this to be static?

Reply:
It doesn't have to be static, should I change it?

Reply:
I expect you want to decorate it. yes.

Reply:
done